### PR TITLE
Postgres stable!

### LINKED ISSUE #207: Postgres - data retention/backup file
- Recreate backup script?

### PR DESCRIPTION
**Pull request by Rock-it-science**
Migration to postgres! Closes #207 & #208.
There are a ton of moving parts here so let me know if you have any difficulties or find any problems.
The only major change to the docker setup process is now on your first-time build you should start the database container on its own (or start both containers and let the web one crash) as postgres needs to build, and Django won't wait for that to happen. So run the database, then run the web server. You only have to do this the first time you build as the files will be persisted to your computer (but not the repo! More on that below). I updated most of the documentation, so you can follow the instructions there, and feel free to let me know or push your own commit if I missed anything.
The other major change is that for persisting data to the repo, we are using dump files through django instead of through the DBMS. I added instructions for that in the readme, but it is a little simpler than before.
I may have to add a couple extra commits for the CI script as that workflow has changed.

### CODE REVIEW

**Comment by Rock-it-science** [comment]
Also! Your env file should look something like this now (some of these fields are not necessary, but I will have to go through and figure out which):
```MSSQL_DB_NAME=lims_db
MSSQL_HOST=lims_db_server_mssql
DB_USER=root
DB_PASSWORD=Lab_rats2021
SECRET_KEY=<same as before, specific to your machine>

POSTGRES_DB=lims_db
POSTGRES_HOST=lims_db_server_postgres
POSTGRES_USER=root
POSTGRES_PASSWORD=Lab_rats2021```

File: README.md [hunk 1]
```diff
@@ -103,47 +103,19 @@ The [testing README](LIMS_IMAGE/web/tests/README.md) is found in the testing fol
 
 1. Ensure docker and docker-compose are installed, and that this repo is cloned to your machine.
 1. cd into `LIMS_IMAGE`.
-1. Run `docker-compose up --build`.
+1. For first-time setup run `docker-compose up --build db-postgres` to let the database build (make sure you wait until it says the database is ready to accept connections. It may pause after the message 'ok' but that does not mean it is finished).
```
Comment (shepkeira) [inline README.md:106]: for first time, do you mean, that now that i've run this command the first time, I don't need to run it again?

**Review by shepkeira — APPROVED** [review]

**Comment by shepkeira** [comment]
It works! Amazing!

File: README.md (hunk 1 above)
Comment (Rock-it-science) [reply README.md:106]: Yes, unless you delete the postgres data folder

**Pull request merged**